### ISSUE TITLE
Timestamping not usefull for conditionally acting on delta

### ISSUE THREAD

**Issue by baskapteijn** [Erriez/ErriezTimestamp#4]
The current library is most useful for logging how long something took to execute to the console. However, this is my use case in pseudo code (that doesn't work with the library):

```
TimestampMillis timestamp;

void setup()
{
  timestamp.start();
}

void loop() {
  timestampDelta = timestamp.end();  <-- This overwrites the internal start, which causes it not to work
  if (timestampDelta >= 15000) {
    ...
    timestamp.start();
  }
}
```
Possible solution:

1. Most logical would be to not overwrite the internal .start() variable until .start() is called again.

So maybe its even better to make the class act like this:
start(): to store a start internally
delta() : to return the delta of millis() - start
print() : calls delta() and prints the delta. Optionally you can keep it returning the delta, which you do now.

**end() is not required**, application can end it just by calling start again or keep it rolling as long as wanted.

This is simple and requires no protection.

**Comment by Erriez** [comment]
Thanks for your feedback. What do you think of this proposal in a new branch "feature-delta-time"?

https://github.com/Erriez/ErriezTimestamp/tree/feature-delta-time

**Comment by baskapteijn** [comment]
Almost there! See review comment(s).

**Comment by Erriez** [comment]
Hmm, I can't find your review comment(s). According to the GitHub documentation, it is only possible to add comments on a Pull request. Or did I miss something?

**Comment by baskapteijn** [comment]
Added review comments on the last conmit of your branch i think.

**Comment by Erriez** [comment]
Found it and created a [Pull request #6](https://github.com/Erriez/ErriezTimestamp/pull/6) for discussion.

**Comment by Erriez** [comment]
Fixed in commit 992c914.
